.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Please follow our code of conduct when engaging in the beman community:
+https://github.com/bemanproject/beman/blob/main/docs/CODE_OF_CONDUCT.md
+-->
+
+<!--
+Thank you for your contribution!
+
+If you are updating project structure or build configs:
+- Make sure your contribution does not go against the beman standard:
+  https://github.com/bemanproject/beman/blob/main/docs/BEMAN_STANDARD.md
+- For new CMake arguments/ presets: please make sure you added appropriate CI tests.
+
+If you are updating documentations:
+- Make sure badges and pictures does not impact readability.
+
+If you are updating any implementation details:
+- Make sure you submit appropriate testing.
+
+We encourage small & incremental additions instead of large redesigns.
+They are easier and faster to review.
+They are also less likely to introduce bugs.
+
+While we do not formally adopt this guide as a standard,
+we encourage you to read and consider:
+"The CL author’s guide to getting through code review".
+https://google.github.io/eng-practices/review/developer/
+
+Regardless, feel free to open a PR on your existing changes.
+We appreciate the suggestion and will help out.
+
+Please run pre-commit against your change to comply with our linting rules.
+The command to check all files in the directory is:
+pre-commit run --all-files
+-->
+
+<!-- markdownlint-disable-next-line MD041 -->
```

Comment:
why is this needed?

Reply:
sure -- well top level headings are sometimes renderer as too large and overbearing -- of course css can in theory fix that.  No matter what, I find it disappointing that the exemplar repo has to silence the checking tool

Reply:
Yeah I do agree the current checking tool is very pedantic sometimes.